roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -2,29 +2,44 @@
 ##Plausible TODOs:
 # Install Helvetica Neue and Consolas from ngi_reports/data/pandoc_templates/assets/fonts/
 
-- name: Create TeXLive destination 
+- stat: path="{{ texLive_dest }}/tlpkg/texlive.profile"
+  register: texlive_installed
+
+- name: Create TeXLive destination
   file: path={{ texLive_dest }} state=directory mode=g+rwxs
   tags: ngi_reports
 
-- name: Download TeXLive
+- name: Download TeXLive (Skipped if installed)
   get_url:
     url: "{{ texLive_url }}"
-    dest: "{{ texLive_dest }}/{{ texLive_file }}"
+    dest: "{{ texLive_dest }}/src/{{ texLive_file }}"
+    when: not texlive_installed
```

Comment:
I don't think you would have to worry about this, as the `force` argument to `get_url` is by default `no`. I.e. the file will only be downloaded if the destination does not exist. It is a good solution with the `stat` on L5 otherwise.